lighthouse-core/test/audits/seo/meta-description-test.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Audit = require('../../../audits/seo/meta-description.js');
+const assert = require('assert');
+
+/* eslint-env mocha */
+
+describe('SEO: description audit', () => {
+  it('fails when HTML does not contain a description meta tag', () => {
+    const auditResult = Audit.audit({
+      MetaDescription: null
+    });
+    assert.equal(auditResult.rawValue, false);
+    assert.ok(auditResult.debugString);
```

Comment:
it would probably be (slightly) better to check which string is getting returned as the `debugString` since there's two different cases (could go for an exact string match or a `debugString.includes()` on `'empty'` and `'missing'`

Reply:
+1 to a quick includes check